yarn.lock

```diff
@@ -1803,206 +1803,115 @@
 
 "@contentful/app-sdk@4.21.1", "@contentful/app-sdk@^4.17.1", "@contentful/app-sdk@^4.21.1", "@contentful/app-sdk@^4.6.0":
   version "4.21.1"
-  resolved "https://registry.yarnpkg.com/@contentful/app-sdk/-/app-sdk-4.21.1.tgz#500ebc69d0a6a6cff5be888aee6cf11e55d602ae"
+  resolved "https://npm.pkg.github.com/download/@contentful/app-sdk/4.21.1/500ebc69d0a6a6cff5be888aee6cf11e55d602ae#500ebc69d0a6a6cff5be888aee6cf11e55d602ae"
   integrity sha512-sRaWKa18x3EWH84qYHT3W/yXsYKqijgPJgQhGZUcdX9Y3d38ZNOikt/FopE399s/MqyyfVJcM+CFHSC0JczAEw==
 
 "@contentful/browserslist-config@^3.0.0":
   version "3.0.0"
-  resolved "https://registry.yarnpkg.com/@contentful/browserslist-config/-/browserslist-config-3.0.0.tgz#e1cc8bc1cecaec9406633ffaf600fb489a204273"
+  resolved "https://npm.pkg.github.com/download/@contentful/browserslist-config/3.0.0/e1cc8bc1cecaec9406633ffaf600fb489a204273#e1cc8bc1cecaec9406633ffaf600fb489a204273"
   integrity sha512-e2OQlbOHMPZCI6BMA3OkTgOszPs2dhkbm+EXlllhBqmIvCOzLuA5iOGfoCEddkCqM8VHI+Zhdumq8XnFtSF+Jw==
 
 "@contentful/contentful-slatejs-adapter@^15.13.1", "@contentful/contentful-slatejs-adapter@^15.16.5":
   version "15.16.12"
-  resolved "https://registry.yarnpkg.com/@contentful/contentful-slatejs-adapter/-/contentful-slatejs-adapter-15.16.12.tgz#aa65e092a15877d9ca9b8adcaa0d577e176f05be"
+  resolved "https://npm.pkg.github.com/download/@contentful/contentful-slatejs-adapter/15.16.12/aa65e092a15877d9ca9b8adcaa0d577e176f05be#aa65e092a15877d9ca9b8adcaa0d577e176f05be"
```

Comment:
There should be no reference to github packages in this repo. See [here](https://contentful.slack.com/archives/C03ED2VDZB3/p1708434900309169?thread_ts=1708428664.612819&cid=C03ED2VDZB3)